hack/lib/integration.sh

```diff
@@ -52,11 +52,11 @@ function os::integration::configresolver::start() {
 
     os::log::info "Starting the config resolver..."
     ci-operator-configresolver --config "${config}"       \
+                               --address ":8088"          \
```

Comment:
somehow `--address` did not get parsed when it was at the end of the flag list?

Reply:
we need to use somethiong *other* than `:8088` as `boskos` only knows how to serve on that port